docs/guides/advanced/data/4844_data.md

```diff
@@ -0,0 +1,185 @@
+
+## The Change with 4844 Blob Transactions
+
+In January 2024, the introduction of 4844 blobs brought several significant changes:
+
+- Blob transactions have their own fee market. This means the cost of using blobs can be different from the cost of
+  using gas.
+- Users must pay for a whole blob, which is 128kb, regardless of how much data they actually use. This differs from
+  calldata, where payment was based on the amount of data sent.
+- The contents of the blobs are not directly accessible. Instead, they contain unique KZG commitments and precompiles.
+
+**In the first version, we will be using up to 2 blobs per batch - so 256kb of pubdata**
+
+In this article, we will mainly discuss how we adapted our system, including our proving system, to work with these KZG
+commitments.
+
+**Why did Ethereum introduce these commitments instead of continuing with the previous hash-based system?**
+
+The short answer is, that while the current version of Ethereum doesn't gain much from this change, the next version,
+known as 'full Danksharding,' will greatly benefit from these commitments, as they will allow the addition of many more
+blobs.
+
+## Technical details
+
+Let's go back to our problem statement:
+
+**We have 128kb of data in L2, and we want to prove on L1 that it was correctly 'attached' to L1 (that is - that it was
+made available).**
+
+This verification is done in 14 steps, divided into three parts: inside the prover, inside the state keeper, and finally
+within the L1 contract.
+
+### Inside the prover
+
+Most of this is done in either witness generator or a new circuit type 4844:
+
+| step                       | explanation                                                                                                                                                                                                                                  |
+| -------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Split data                 | Divide the data into 4096 units of 256-bit points.                                                                                                                                                                                           |
+| Create Polynomial          | Form a polynomial `p` that passes through these points. For more clarity on this, refer to [Blobspace 101][blobspace_101]                                                                                                                    |
+| Trusted setup              | Obtain the [trusted setup][trusted_setup] from Ethereum's KZG ceremony (also known as the secret point 's').                                                                                                                                 |
```

Comment:
```suggestion
| Trusted Setup              | Obtain the [trusted setup][trusted_setup] values from Ethereum's KZG ceremony (also known as the secret point 's').     
```
I'm not so sure about "the secret point 's'" -- the secret point is secret, this is not what you get from the ceremony.

Reply:
I don't see a reason to explain it at all. I'm not even sure that "the secret point 's'" is _known_. You can probably replace with CRS/SRS if you want.
Note also that it's not Atheneum's ceremony, but Aztec's (https://aztec.network/blog/aztec-crs-the-biggest-mpc-setup-in-history-has-successfully-finished/).